src/commands/draft/draftCommands.ts

```diff
@@ -229,13 +230,12 @@ async function getRepo(octokit: Octokit, remote: Remote): Promise<GitHubRepo | n
         return null;
     }
 
-    const [owner, repo] = url
-        .replace(/\.git$/, "")
-        .split("/")
-        .slice(-2);
+    const parts = url.replace(/\.git$/, "").split(/[:/]/); //Split on both : and /, Removes .git
+    const [owner, repo] = parts.slice(-2);
+
     let response: RestEndpointMethodTypes["repos"]["get"]["response"];
     try {
-        response = await octokit.repos.get({ owner, repo });
+        response = await octokit.rest.repos.get({ owner, repo });
```

Comment:
💡 This is interesting: minor quesiton please: Can you please add doc as to why `octokit.repos.get(`  worked in same way as `octokit.rest.repos.get(` worth to archive in case this comes in any future discussion please. Thank you!!

Reply:
Good catch.

When debugging the bug, I could not get the original "octokit.repos.get(" to function. This was an older syntax used in [octokit v17](https://octokit.github.io/rest.js/v17l) and older.

The [current version](https://octokit.github.io/rest.js/v20) standard uses the rest namespace. It may be that when implemented previously an older version of the octokit was being utilized.